libs/libc/stream/lib_blkoutstream.c

```diff
@@ -217,6 +227,7 @@ int lib_blkoutstream_open(FAR struct lib_blkoutstream_s *stream,
     }
 
   stream->inode        = inode;
+  stream->public.put   = blkoutstream_putc;
```

Comment:
@xiaoxiang781216 maybe stream method also should be `putc`? What do you think? I've seen that recently you done some clean-up and changes from `putc` to `put`, but do not know the reasons why we stay with `put` now

Reply:
It's little bit inconsistent:

1. The user see the function as lib_stream_put or lib_outstream_s::put
2. The implementation normally name xxx_putc as statical function